### PR TITLE
More histogram facet styling

### PR DESCRIPTION
**Pull request by melissachang**
- Update bar and header to use Terra colors from Lou's screens
- Fixed bar width
- Create FacetHeader component and use it in both TextFacet and HistogramFacet

### CODE REVIEW

File: ui/src/components/facets/FacetHeader.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        </Typography>
+      </React.Fragment>
+    );
+  }
+
+  /**
+   * @param facetValues FacetValue[] to sum counts over
+   * @param selectedValueNames Optional string[] to select a subset of facetValues to sum counts for
+   * @return number count the total sum of all facetValue counts, optionally filtered by selectedValueNames
+   * */
```
Comment (bfcrampton) [inline ui/src/components/facets/FacetHeader.js]: micro nit: remove extra `*`?

File: ui/src/components/facets/FacetHeader.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  sumFacetValueCounts(facetValues, selectedValueNames) {
+    let count = 0;
+    if (selectedValueNames == null || selectedValueNames.length === 0) {
+      facetValues.forEach(value => {
```
Comment (bfcrampton) [inline ui/src/components/facets/FacetHeader.js]: nit: do you need the brackets?

```
facetValues.forEach(value => count += value.count);
```

File: ui/src/components/facets/FacetHeader.js (hunk 1 above)
Comment (melissachang) [reply ui/src/components/facets/FacetHeader.js]: Done

File: ui/src/components/facets/FacetHeader.js (hunk 2 above)
Comment (melissachang) [reply ui/src/components/facets/FacetHeader.js]: Maybe not, but I think this is cleaner -- it matches 64-66

**Pull request merged**